Azure/sonic-buildimage
# [centec] support TM.MX board v682-48y8c_d

Adding platform support for centec v682-48y8c_d.



#### Why I did it
Adding platform support for centec v682-48y8c_d.
This switch has 48 SFP+ (1G/10G/25G) ports, 8 QSFP28 (40G/100G) ports on CENTEC TsingMa.MX.
CPU used in v682-48y8c_d is Intel(R) Xeon(R) CPU D-1527. 

#### How I did it
1. Modify related code in platform and device directory.
2. Upgrade sai to v1.8.
3. fix bug for parallel compile centec platform modules.

#### How to verify it
Build centec amd64 sonic image, verify platform functions (port, sfp, led etc) on centec v682-48y8c_d board.

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
